ci-operator/step-registry/upi/conf/vsphere/platform-external/upi-conf-vsphere-platform-external-commands.sh

```diff
@@ -189,6 +190,15 @@ status:
 EOF
 yq-go m -x -i "manifests/cluster-infrastructure-02-config.yml" "/tmp/infraPlatformSpecPatch.yml"
 
+else 
+  if [[ ! -z ${PLATFORM_NAME} ]]; then
+    DERIVED_PLATFORM_NAME=$(cat "inst3/manifests/cluster-infrastructure-02-config.yml" | yq-go '.spec.platformSpec.external.platformName')
```

Comment:
maybe this could be the path used in the step:

```suggestion
    DERIVED_PLATFORM_NAME=$(cat "manifests/cluster-infrastructure-02-config.yml" | yq-go '.spec.platformSpec.external.platformName')
```